pkgs/applications/graphics/akira/meson-post-install.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/meson.build b/meson.build
+index 9a4ad56..ce6c003 100644
+--- a/meson.build
++++ b/meson.build
+@@ -65,5 +65,3 @@ add_global_arguments(['-DG_LOG_DOMAIN="Akira"'], language:'c')
+ subdir('src')
+ subdir('data')
+ subdir('po')
+-
+-meson.add_install_script('build-aux/meson/post_install.py')
```

Comment:
Please don't do this. What you probably need to do in the expression is add
`desktop-file-utils` and `python3` to `nativeBuildInputs` and patch the shebang https://github.com/NixOS/nixpkgs/blob/4e8c2f08b01486f2de0c7eb2468932e94b28030c/pkgs/desktops/pantheon/apps/elementary-code/default.nix#L86.